packages/core-contracts/contracts/interfaces/IERC20.sol

```diff
@@ -2,6 +2,9 @@
 pragma solidity ^0.8.0;
 
 interface IERC20 {
+    error InsufficientAllowance(uint required, uint existing);
+    error InsufficientBalance(uint required, uint existing);
```

Comment:
should events also be in interfaces?  not sure whats the standard here and the reasons behind it

Reply:
Yes I think so. Events in interfaces should help subgraph developers.